slither/printers/summary/printer_human_summary.py

```diff
@@ -0,0 +1,102 @@
+"""
+Module printing summary of the contract
+"""
+import logging
+
+from slither.detectors.detectors import Detectors
+from slither.printers.abstractPrinter import AbstractPrinter
+from slither.utils.code_complexity import compute_cyclomatic_complexity
+from slither.utils.colors import green, red, yellow
+
+
+class PrinterHumanSummary(AbstractPrinter):
+    ARGUMENT = 'print-human-summary'
+    HELP = 'Print an human readable summary of the contracts'
+
+    @staticmethod
+    def get_summary_erc20(contract):
+        txt = ''
+        functions_name = [f.name for f in contract.functions]
```

Comment:
```suggestion
        functions_name = [f.name.lower() for f in contract.functions]
```

Reply:
I am not 100% sure about it but we should consider either converting things to lowercase or matching some of the strings as case insensitive.

I assume that for example `paused` function is not a standard one (e.g. for erc20 token) and one can name it as `isPaused` or `getPaused` (lol).